src/gucs.rs

```diff
@@ -30,6 +31,14 @@ pub(crate) fn init() {
         &PLRUST_PG_CONFIG,
         GucContext::Sighup,
     );
+
+    GucRegistry::define_string_guc(
+        "plrust.tracing_level",
+        "The tracing level to use while running pl/rust",
+        "The tracing level to use while running pl/rust. Should be `error`, `warn`, `info`, `debug`, or `trace`",
+        &PLRUST_TRACING_LEVEL,
```

Comment:
We could, alternatively, allow the user to pass a set of tracing directives.

Reply:
Neither jumps out at me as obviously preferable so this is probably fine.